dashboard/app/helpers/levels_helper.rb

```diff
@@ -39,7 +39,7 @@ def url_from_path(path)
 
   def set_videos_and_blocks_and_callouts
     @autoplay_video_info = select_and_track_autoplay_video
-    @callouts = select_and_remember_callouts(@script_level.nil?)
+    @callouts = select_and_remember_callouts(params[:show_callouts])
```

Comment:
The `/levels/n` pages will no longer show callouts by default, but I think this is the right fix.  We're moving away from `/levels/n` being admin-only so it makes sense for it to behave the same as other pages that do have `@script_level` defined.